multidim-interop/composer/index.ts

```diff
@@ -0,0 +1,127 @@
+import sqlite3 from "sqlite3";
+import { open } from "sqlite";
+import { promisify } from "util";
+import { TestPlan, TestPlans } from "dsl/src/dsl";
+import { createHmac } from "crypto";
+import { Version } from "../versions";
+
+
+export async function buildTestplans(versions: Array<Version>): Promise<TestPlans> {
```

Comment:
It would be nice to have a test for this.

Reply:
I don't understand. This whole thing is the test, isn't it?